docs/user-guide/debugging-services.md

```diff
@@ -555,6 +555,25 @@ There are three things to check:
   Engine doesn't do port remapping, so if your application serves on 8080,
   the `containerPort` field needs to be 8080.
 
+### A Pod cannot reach itself via Service IP
+
+This mostly happens in `iptables` mode. There is `--hairpin-mode` flag in `Kubelet`.
+In order to allow endpoints of a Service to loadbalance back to themselves
+if they should try to access their own Service, `--hairpin-mode` flag has to be set
+to `haripin-veth` or `promiscuous-bridge`.
+
+The common steps to trouble shoot this are as follows:
+
+* Ensure `--hairpin-mode` is set to `haripin-veth` or `promiscuous-bridge`.
+
+* Check if there is any log lines with key word `hairpin` in kubelet.log. The log is usually
```

Comment:
what about if it's none? (impact, and how to fix, or link to doc, or ask them to contact us or something)